tools/src/main/java/org/apache/kafka/trogdor/workload/ExternalCommandWorker.java

```diff
@@ -389,7 +389,7 @@ public void stop(Platform platform) throws Exception {
             spec.shutdownGracePeriodMs().get() : DEFAULT_SHUTDOWN_GRACE_PERIOD_MS;
         if (!executor.awaitTermination(shutdownGracePeriodMs, TimeUnit.MILLISECONDS)) {
             terminatorActionQueue.add(TerminatorAction.DESTROY_FORCIBLY);
-            executor.awaitTermination(1000, TimeUnit.DAYS);
+            executor.awaitTermination(1, TimeUnit.DAYS);
```

Comment:
1000 days is overkill, 1 day is consistent with the other workers

Reply:
In general if this returns and the executor hasn't terminated, we have leaked some threads.  I agree one day should be enough to shut down, but if we actually hit that timeout, there is another bug we have to fix.